broker/core/src/main.cc

```diff
@@ -62,7 +63,9 @@ using namespace com::centreon::exceptions;
 
 std::shared_ptr<asio::io_context> g_io_context =
     std::make_shared<asio::io_context>();
-bool g_io_context_started = false;
+
+std::unique_ptr<com::centreon::common::pool>
+    com::centreon::common::pool::_instance;
 
```

Comment:
Would be better in pool.cc

Reply:
tried and I have noticed duplication of symbols